decidim-proposals/app/commands/decidim/proposals/admin/update_proposal.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      #  A command with all the business logic when an admin updates a proposal.
+      class UpdateProposal < Rectify::Command
```

Comment:
Since this is only used to change the category I'd rename it to `UpdateProposalCategory` or something similar

Reply:
Ok, will rename.